server/datastore/mysql/migrations/tables/20240322145615_CreateTableNanoDDMRequests.go

```diff
@@ -0,0 +1,41 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240322145615, Down_20240322145615)
+}
+
+func Up_20240322145615(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE mdm_apple_declarative_requests (
+  id BIGINT NOT NULL AUTO_INCREMENT,
```

Comment:
We usually default to `int unsigned` unless we think there'll be huge churn (I think `software` is one of the only PKs that is BIGINT?). If we think that's the case here, would be good to add a comment for the bigint rationale.

Reply:
There's no reason I used bigint as opposed to int, I can make that change

Reply:
It makes a new entry for each DDM request, so I suppose on a large deployment the table will quickly grow, but not likely fast enough to warrant a bigint